inc/remove-admin-menu-items.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Remove admin menu pages.
+ *
+ * @package MEOM Dodo
+ */
+
+namespace MEOM\Dodo;
+
+/**
+ * Remove admin menu items.
+ */
+function remove_admin_menu_items() {
+    /**
+     * Filters when to show all admin menu items.
+     *
+     * By default show everything when not on production.
+     *
+     * @param array $show_all_admin_menu_items When to show all admin menu items.
+     */
+    $show_all_admin_menu_items =
```

Comment:
We should remove this check and rely only on user profile setting. In that way it's easier to share local environment screen with or without all admin menu items quickly.

cc: @giraffewoods

Reply:
Removed in c6507dc. I'm going to merge this PR so we can move forward with user setting.